backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+include::_attributes/common-attributes.adoc[]
```

Comment:
We should reorganize this metadata info so that it follows OCP guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata

```suggestion
:_content-type: ASSEMBLY
[id="oadp-introduction"]
= Introduction to {oadp-full}
include::_attributes/common-attributes.adoc[]
:context: oadp-api
:namespace: openshift-adp
:local-product: OADP

toc::[]
```